apps/rush-lib/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@aws-sdk/client-s3": "~3.3.0",
```

Comment:
@iclanton these `@aws-sdk` dependencies bring in 3,371files (16MB uncompressed).  The Azure stuff also brought in a bunch of dependencies.  Maybe it's time to consider an install-on-demand facility for Rush.